contents/docs/posthog-ai/pricing-faq.mdx

```diff
@@ -0,0 +1,61 @@
+# PostHog AI credits
+
+PostHog uses **AI credits** to charge for all of its AI features.  
+Each time you interact with an AI-powered feature — like asking PostHog to write SQL, summarize data, or analyze sessions — credits are used based on the effort required to complete your request.
```

Comment:
We use line wrapping in the editor with `cmd + z` in vscode instead of manual next line positions


```suggestion
AI features in PostHog consume **AI credits**. Each time you interact with an AI-powered feature – like asking PostHog to write SQL, summarize data, or analyze sessions – credits are used based on the effort required to complete your request.
```

This is also probably a little annoying but we use `–` (en dashes) 😅